tests/components/modbus/test_init.py

```diff
@@ -329,11 +329,6 @@ async def test_ok_struct_validator(do_config) -> None:
             CONF_VIRTUAL_COUNT: 2,
             CONF_DATA_TYPE: DataType.INT32,
         },
-        {
```

Comment:
This is wrong, you remove tests ?

Could it be you work on an older dev ?

Reply:
This is a duplicate of the test in line 315–319